pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -19,16 +19,16 @@ const (
 	// AccountStatusClaimed const for Claimed status
 	AccountStatusClaimed AccountStateStatus = "Claimed"
 	// AccountStatusTransfering const for Transfering status
-	accountStatusTransfering AccountStateStatus = "Transfering"
+	accountStatusTransfering AccountStateStatus = "Transfering" //nolint:varcheck,deadcode,unused // May be used in future
 	// AccountStatusTransfered const for Transfering status
-	accountStatusTransfered AccountStateStatus = "Transfered"
+	accountStatusTransfered AccountStateStatus = "Transfered" //nolint:varcheck,deadcode,unused // May be used in future
 	// AccountStatusDeleting const for Deleting status
-	accountStatusDeleting AccountStateStatus = "Deleting"
+	accountStatusDeleting AccountStateStatus = "Deleting" //nolint:varcheck,deadcode,unused // May be used in future
 	// AccountStatusPendingVerification const for Pending Verification status
-	accountStatusPendingVerification AccountStateStatus = "PendingVerification"
+	accountStatusPendingVerification AccountStateStatus = "PendingVerification" //nolint:varcheck,deadcode,unused // Maybe be used in future
```

Comment:
It's odd to preserve these as private+unused. If we ever were to use them in the future, we would export them to match `AccountStatusRequested` and `AccountStatusClaimed`, right?

IMO it would make sense to replace all of this with just a comment listing the `AccountStateStatus`es that we "might" use in the future.

But if we are going to keep them, we should make the case in the docstrings match.

Reply:
I struggled with that decision.  I didn't want to leave unused code, but felt like that made for a complete list of states. I'll remove them.  Thanks for the input.